django_cloud_deploy/tests/unit/crash_handling/crash_handling_test.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit test for django_cloud_deploy/crash_handling."""
+
+import unittest
+from unittest import mock
+from urllib import parse
+
+from django_cloud_deploy import crash_handling
+from django_cloud_deploy.cli import io
+
+
+@mock.patch('tempfile.mkstemp', return_value=(1, '/tmp/tmpfile'))
```

Comment:
This will will really get written to, right? Do you need to mock this?

Reply:
This will not. Because I also mocked os.fdopen, nothing will be written to this path. I was just worrying about creating a new file each time this test case is ran, so I mocked it.